bokehjs/src/coffee/models/mappers/color_mapper.coffee

```diff
@@ -24,17 +24,19 @@ export class ColorMapper extends Transform
   v_map_screen: (data, image_glyph=false) ->
     values = @_get_values(data, @_palette, image_glyph)
     buf = new ArrayBuffer(data.length * 4)
-    color = new Uint32Array(buf)
-
     if @_little_endian
+      color = new Uint8Array(buf)
       for i in [0...data.length]
         value = values[i]
-        color[i] =
-          (0xff << 24)                   | # alpha
-          ((value & 0xff0000) >> 16)     | # blue
-          (value & 0xff00)               | # green
-          ((value & 0xff) << 16);          # red
+        ind = i*4
+        # Bitwise math in JS is limited to 31-bits, to handle 32-bit value
+        # this uses regular math to compute alpha instead (see issue #6755)
+        color[ind] = Math.trunc((value/4278190080.0) * 255)
```

Comment:
This may be actually a solvable problem in typescript, because we can provide our own typings for the standard library, and explicitly say what we expect to be implemented by all supported runtimes. This way using `Math.trunc` would results in an error (e.g. "`Math` namespace doesn't implement `trunc` function").